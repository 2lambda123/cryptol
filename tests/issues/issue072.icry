:set show-examples=false
:sat \x -> x > 0x4
:t it
:set satNum = all
:sat \x -> x > 0x4
:t it
:set satNum = foobar
:set satNum = 0
:set satNum = 3
:sat \x -> x > 0x4
:t it
