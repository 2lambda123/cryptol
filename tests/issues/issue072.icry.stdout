Loading module Cryptol
it : {result : Bit, arg1 : [4]}
it : [11]{result : Bit, arg1 : [4]}
must be an integer > 0 or "all"
must be an integer > 0 or "all"
it : [3]{result : Bit, arg1 : [4]}
